examples/crm/src/lib.rs

```diff
@@ -163,10 +163,17 @@ impl Component for Model {
             },
             Scene::NewClientForm(ref client) => html! {
                 <div class="crm">
+                    <h1>{"Add new client"}</h1>
```

Comment:
Ideally there would be an indefinite adjective here
```suggestion
                    <h1>{"Add a new client"}</h1>
```

Reply:
Done